.github/workflows/build.yaml

```
@@ -44,10 +44,79 @@ jobs:
+      - build-plugins
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Download plugins
+        uses: actions/download-artifact@v4
+        with:
+          path: Assets/Plugins
+          merge-multiple: true
+
+      - name: Extract plugins
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2035:info:2:6: Use ./*glob* or -- *glob* so names with dashes won't become options

(shellcheck)